modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/MarkedItems/IMarkedItemPublicAppService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using Volo.Abp.Application.Services;
+
+namespace Volo.CmsKit.Public.MarkedItems;
+
+public interface IMarkedItemPublicAppService : IApplicationService
+{
+    Task<MarkedItemWithToggleDto> GetForToggleAsync(string entityType, string entityId);
```

Comment:
The name `GetForToggleAsync` is not good enough. How about making it simple like `GetAsync` or `GetForUserAsync`?